### DISCUSSION TITLE
Split long long config lines?

### DISCUSSION THREAD

**Discussion by pschmitt** [hyprwm/Hyprland#2284] [Q&A] [answered]
As the title says, is there any way to split long lines in Hyprland's config?

Here's an example:

```conf
exec = swayidle -w timeout 300 'exec $lock' timeout 600 'hyprctl dispatch dpms off' resume 'hyprctl dispatch dpms on' before-sleep 'exec $lock'
```

**Comment by vaxerski** [comment, marked as answer]
not at the moment

**Reply by azizsayfiddin** [reply]
Now?

**Reply by vaxerski** [reply]
if I am not mistaken putting a \ at the end of a line should work

**Reply by azizsayfiddin** [reply]
https://github.com/user-attachments/assets/e2c6e1b7-a0f8-42e3-b1b3-e4158541744c

**Reply by vaxerski** [reply]
ah, right, it's an open MR where the autor has gone MIA. See https://github.com/hyprwm/hyprlang/pull/51
